spec/features/participants/happy_validation_journeys_for_ect_fip_spec.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require_relative "./participant_validation_steps"
+
+RSpec.feature "ECT participant validation journey for FIP induction", type: :feature, js: true, rutabaga: false, with_feature_flags: { participant_validation: "active" } do
```

Comment:
You will be glad to know that latest develop should not have any rutabaga tests - and we axed the gem too. So I hope we don't need to set it to false anymore.